plugins/GSdx/GSRendererHW.cpp

```diff
@@ -201,22 +201,23 @@ void GSRendererHW::InvalidateLocalMem(const GIFRegBITBLTBUF& BITBLTBUF, const GS
 	m_tc->InvalidateLocalMem(m_mem.GetOffset(BITBLTBUF.SBP, BITBLTBUF.SBW, BITBLTBUF.SPSM), r);
 }
 
-int GSRendererHW::Interpolate_UV(float alpha, int t0, int t1)
+uint16 GSRendererHW::Interpolate_UV(float alpha, int t0, int t1)
 {
 	float t = (1.0f - alpha) * t0 + alpha * t1;
-	return (int)t & ~0xF; // cheap rounding
+	return (uint16)t & ~0xF; // cheap rounding
 }
 
+//X1 not even used?? Should this be similar to alpha1 with different rounding?
 float GSRendererHW::alpha0(int L, int X0, int X1)
 {
-	float x = (X0 + 15) & ~0xF; // Round up
-	return (x - X0) / (float)L;
+	int x = (X0 + 15) & ~0xF; // Round up
+	return float(x - X0) / (float)L;
 }
```

Comment:
It probably would if I put the warning level as less tolerant.
It could be removed, I might still do.  I'm guessing it's there if needed in the future or for easier debugging? I dunno.